src/admin/components/views/Dashboard/Default.tsx

```diff
@@ -41,24 +42,28 @@ const Dashboard: React.FC<Props> = (props) => {
 
   useEffect(() => {
     setGroups(groupNavItems([
-      ...collections.map((collection) => {
-        const entityToGroup: EntityToGroup = {
-          type: EntityType.collection,
-          entity: collection,
-        };
+      ...collections
+        .filter(({ admin: { hidden } }) => !(typeof hidden === 'function' ? hidden({ user }) : hidden))
```

Comment:
Any reason this isn't a `!==` ?

Reply:
Not sure I follow... `hidden` could be undefined, a function or a boolean. I'm not sure I can make this more succinct.